File: findMostFrequentVowelConsonant.py
from collections import Counter

class Solution:
    def maxFreqSum(self, s: str) -> int:
        vowels = {'a', 'e', 'i', 'o', 'u'}
        freq = Counter(s)
        
        max_vowel = max((freq[char] for char in vowels if char in freq), default=0)
        max_consonant = max((freq[char] for char in freq if char not in vowels), default=0)
        
        return max_vowel + max_consonant

File: findMostFrequentVowelConsonant.go
func maxFreqSum(s string) int {
    freq := make(map[rune]int)
    vowels := map[rune]bool{'a': true, 'e': true, 'i': true, 'o': true, 'u': true}
    
    for _, ch := range s {
        freq[ch]++
    }
    
    maxVowel := 0
    maxConsonant := 0
    
    for ch, count := range freq {
        if vowels[ch] {
            if count > maxVowel {
                maxVowel = count
            }
        } else {
            if count > maxConsonant {
                maxConsonant = count
            }
        }
    }
    
    return maxVowel + maxConsonant
}
